src/Package.zig

```diff
@@ -575,8 +575,12 @@ fn fetchAndUnpack(
             });
         }
 
-        const content_type = req.response.headers.getFirstValue("Content-Type") orelse
+        var content_type = req.response.headers.getFirstValue("Content-Type") orelse
             return report.fail(dep.url_tok, "Missing 'Content-Type' header", .{});
+        // Media type _may_ be followed by semicolon delimited parameters we don't care about
+        // https://www.rfc-editor.org/rfc/rfc9110#media.type
+        content_type = std.mem.splitScalar(u8, content_type, ';').first();
+        content_type = std.mem.trim(u8, content_type, " ");
```

Comment:
I would say the "Content-Type" header is what contains a "media type" (formerly called "MIME type") and maybe a charset and a boundary. So I would suggest this instead:
```suggestion
        const content_type = req.response.headers.getFirstValue("Content-Type") orelse
             return report.fail(dep.url_tok, "Missing 'Content-Type' header", .{});
        var content_type_fields = mem.splitScalar(u8, content_type, ';');
        const media_type = content_type_fields.first();
```
and then change `content_type` to `media_type` in the code below.

And I'd change the error message below like this because it no longer prints the whole header:
```zig
        return report.fail(dep.url_tok, "Unsupported 'Content-Type' header media type value: '{s}'", .{media_type});
```

Reply:
Also, the media type should not need to be trimmed.

Reply:
Good suggestions thanks, I think I misread where the space is permitted in the examples here https://www.rfc-editor.org/rfc/rfc9110#field.content-type